arches/app/media/js/views/graph/graph-designer/card-tree.js

```diff
@@ -360,27 +361,23 @@ define([
             },
             selection: selection,
             filter: filter,
-            isFuncNode: function() {
-                var appFuncs = null, appFuncDesc = false, appFuncName = false, nodegroupId = null;
-                if(params.card && this.appliedFunctions()) {
+            isFuncNode: function () {
+                var nodegroupId = null, pdFunction = this.primaryDescriptorFunction;
+
+                // params.card always seems to be undefined...
+                if (params.card && pdFunction) {
```

Comment:
params.card seems to always be undefined. It's not passed as a parameter to the CardTreeViewModel. Not sure if/where this is supposed to happen.

Reply:
I agree - there doesn't look like a case in which params.card has a value. It must have been possible in 2019 (https://github.com/archesproject/arches/commit/82c9e88b9c5e15459910d6a94d6b32557a855ad1).